data_steward/analytics/cdr_ops/vocabulary_upload_qc.py

```diff
@@ -15,27 +15,29 @@
 #
 # # Purpose:
 # This notebook is used to validate new uploads of the OMOP vocabularys prior to implementing in production.<br>
-#
-#
-#  **Link to get the latest custom concepts [HERE](https://github.com/all-of-us/curation/blame/develop/data_steward/resource_files/aou_vocab/CONCEPT.csv).**
-#
 
+# +
 # # + Import packages
 from common import JINJA_ENV
 from utils import auth
 from gcloud.bq import BigQueryClient
 from analytics.cdr_ops.notebook_utils import execute, IMPERSONATION_SCOPES
+
 import pandas as pd
 
+# get current custom concepts
+custom_concepts_csv_directory = ".\..\\..\\resource_files\\aou_vocab\\CONCEPT.csv"
```

Comment:
I would suggest retrieving this resources.AOU_VOCAB_PATH

Reply:
!! Thanks for pointing that out!